vars/Debian.yml

```diff
@@ -1,35 +0,0 @@
----
-# file: vars/Debian.yml
```

Comment:
Deleting this file will break support to others Debian based distributions like most Ubuntu versions.

Best approach is:

Use Debian.yml to support all the latest Debian based distributions.

Add only Debian-Mayorversion.yml or Ubuntu-Mayorversion.yml in case when you need to add compatibility to older versions that have different package names for example.

Reply:
hmm. Lousy of me to delete it ;-) makes no sense.
I usually keep a file for each mayor version anyway in my roles to be explicit for what OS versions it works and I have it tested with.